Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -49,6 +49,7 @@ public IEnumerable<ContentItemTemplate> Templates
         }
 
         public bool LaunchDebugger { get; set; }
+        public bool MultithreadMode { get; set; }
```

Comment:
Indeed. I'll take a look at `Settings` to see how it work. This implies that the setting will be saved between instances, right?